library/core/src/ffi.rs

```diff
@@ -190,7 +194,7 @@ pub struct VaList<'a, 'f: 'a> {
 
     #[cfg(all(
         any(target_arch = "aarch64", target_arch = "powerpc", target_arch = "x86_64"),
-        any(not(target_arch = "aarch64"), not(target_os = "ios")),
+        any(not(target_arch = "aarch64"), all(not(target_os = "macos"), not(target_os = "ios"))),
```

Comment:
Minor style nit `not(any(...))` instead of `all(not(...), not(...))`?

Reply:
Can do! Was just trying to adhere to existing style.

Reply:
Changed!